docs/logbook/LHC/2025_lhc.md

```diff
@@ -39,4 +40,5 @@ The tables below can be sorted by clicking next to the column headers.
 *[NL at injection, IRNL 18cm]: Chroma scans at injection, reference RDT measurements after 8h at inj and with ecloud tunes, arrived at 18cm but dumped soon.
 *[IRNL 1.2m and 18cm]: Validated on/off Momentum at 120cm, and 18cm. Checked sdisp qbump, which did not seem to work. Nonlinear corrections at 18cm: b4, a4 (performed and tested) b3, a3 (only measured/calculated).
 *[Global Corrs at 60cm and 18cm and Validation]: Tested SDisp knobs and did a first squeeze down to 18cm with global corrections calculated and incorporated at 60cm and 18cm. Beam dumped twice because of IR5. Validation only up to 4.5TeV but also couldn't kick high.
+*[Validation of Everything]: Too many k-mods. Validated 120cm, 93cm, 60cm, 44cm, 30cm, 24cm, 18cm. Waistshift and beta*-beating at 18cm. Tried to find better corrections, but dumped before verifying.
```

Comment:
Maybe let's put the "Too many k-mods" a bit later so the tooltip doesn't start with a snarky comment? Or just "Many k-mods" instead?